x11-misc/albert/albert-0.17.5.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..11} )
+
+inherit cmake python-single-r1 xdg-utils
+
+DESCRIPTION="A fast and flexible keyboard launcher"
+HOMEPAGE="https://albertlauncher.github.io/"
+
+PLUGINS_HASH="22881af568d70a3d4c16a901cd49c0e233c14a7a"
+PYTHON_EXTENSIONS_COMMIT="7f571aceaf8e60eee8bb21e1ec4efa0e95523d13"
+JETBRAINS_PYTHON_EXTENSION_COMMIT="b7157473cc923fe4f15023c85a032eeab3627652"
+XKCD_PYTHON_EXTENSION_COMMIT="bf88a964473d65b39c9e09eb48dabb847206f06f"
+
+SRC_URI="
+	https://github.com/albertlauncher/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/albertlauncher/plugins/archive/${PLUGINS_HASH}.tar.gz -> ${PN}-plugins-${PLUGINS_HASH}.tar.gz
+	python-extensions? (
+		https://github.com/albertlauncher/python/archive/${PYTHON_EXTENSIONS_COMMIT}.tar.gz -> ${PN}-python-extensions-${PYTHON_EXTENSIONS_COMMIT}.tar.gz
+		https://github.com/mqus/jetbrains-albert-plugin/archive/${JETBRAINS_PYTHON_EXTENSION_COMMIT}.tar.gz -> ${PN}-jetbrains-python-extension-${JETBRAINS_PYTHON_EXTENSION_COMMIT}.tar.gz
+		https://github.com/bergercookie/xkcd-albert-plugin/archive/${XKCD_PYTHON_EXTENSION_COMMIT}.tar.gz -> ${PN}-xkcd-python-extension-${XKCD_PYTHON_EXTENSION_COMMIT}.tar.gz
+	)
+"
+
+LICENSE="all-rights-reserved"	# unclear licensing #766129
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="debug +python +python-extensions +statistics virtualbox"
+RESTRICT="mirror bindist"
+
+REQUIRED_USE="
+	python-extensions? ( python )
+	python? ( ${PYTHON_REQUIRED_USE} )
+"
+
+RDEPEND="
+	dev-cpp/muParser
+	dev-qt/qtconcurrent:5
+	dev-qt/qtcore:5
+	dev-qt/qtdbus:5
+	dev-qt/qtdeclarative:5
+	dev-qt/qtgraphicaleffects:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtsql:5[sqlite]
+	dev-qt/qtsvg:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtx11extras:5
+	x11-libs/libX11
+	python? ( ${PYTHON_DEPS} )
+	statistics? (
+		dev-qt/qtcharts:5
+		dev-qt/qtxml:5
+	)
+	virtualbox? ( app-emulation/virtualbox[sdk,vboxwebsrv] )
+"
+DEPEND="${RDEPEND}
+	python? ( $(python_gen_cond_dep 'dev-python/pybind11[${PYTHON_USEDEP}]') )"
+
+PATCHES=("${FILESDIR}/${P}-find-and-use-python-libraries-and-headers.patch")
```

Comment:
> * ERROR: x11-misc/albert-0.17.5::gentoo failed (prepare phase):
> *   patch -p1  failed with /var/tmp/portage/x11-misc/albert-0.17.5/files/albert-0.17.5-find-and-use-python-libraries-and-headers.patch

Did you forget to push this?

Reply:
ah yes nevermind, I’m fixing that right now